lms/djangoapps/course_api/blocks/permissions.py

```diff
@@ -14,11 +22,11 @@ def can_access_other_users_blocks(requesting_user, course_key):
     return has_access(requesting_user, 'staff', course_key)
 
 
-def can_access_users_blocks(requested_user, course_key):
+def can_access_self_blocks(requesting_user, course_key):
     """
-    Returns whether blocks for the requested_user is accessible.
+    Returns whether the requesting_user can access own blocks.
     """
     return (
-        (requested_user.id and CourseEnrollment.is_enrolled(requested_user, course_key)) or
-        has_access(requested_user, 'staff', course_key)
+        (requesting_user.id and CourseEnrollment.is_enrolled(requesting_user, course_key)) or
+        has_access(requesting_user, 'staff', course_key)
```

Comment:
Can [CourseStaffRole.ROLE](https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/roles.py#L216) be referenced instead of the string?